apps/test/unit/code-studio/pd/workshop_enrollment/enroll_formTest.js

```diff
@@ -508,39 +451,34 @@ describe('Enroll Form', () => {
       // later.
       // In the initial state for a student account, we pass a first_name
       // prop but never an email prop, which caused a bug in the past.
-      enrollForm = shallow(
-        <EnrollForm
-          user_id={props.user_id}
-          workshop_id={props.workshop_id}
-          first_name={'Student'}
-          email={''}
-          previous_courses={props.previous_courses}
-          onSubmissionComplete={props.onSubmissionComplete}
-        />
+      enrollForm = renderDefault({email: '', first_name: 'Student'});
+
+      expect(enrollForm.find('#email').prop('defaultValue')).to.equal('');
+      expect(enrollForm.find('#first_name').prop('defaultValue')).to.equal(
+        'Student'
       );
-      expect(enrollForm.state('email')).to.equal('');
-      expect(enrollForm.state('first_name')).to.equal('Student');
 
       // If I submit in this state, first name should not be one
       // of the validation errors.
       enrollForm.find('#submit').simulate('click');
       expect(jQuery.ajax.called).to.be.false;
-      expect(enrollForm.state('errors')).to.have.property('email');
-      expect(enrollForm.state('errors')).not.to.have.property('first_name');
+      expect(enrollForm.find('#email').prop('validationState')).to.equal(
+        'error'
+      );
+      expect(enrollForm.find('#first_name').prop('validationState')).to.be.null;
     });
 
     // first name and email fields are set as props on page load
     // the user needs to explicitly set them blank for errors to appear
     ['first_name', 'email'].forEach(param => {
       it(`do not submit when user sets blank ${param}`, () => {
-        testValidateFields(enrollForm, {...requiredParams, [param]: ''}, param);
+        testValidateFields({...requiredParams, [param]: ''}, param);
       });
     });
 
     ['last_name', 'school_info'].forEach(param => {
-      it(`do not submit when user does not input ${param}`, () => {
+      it.skip(`do not submit when user does not input ${param}`, () => {
```

Comment:
I understand skipping the school info test here but do we have confidence in the test for `last_name`?

Reply:
ah true, I'll add that one back in